src/routes/welcome.ts

```diff
@@ -22,12 +23,40 @@ export const createWelcomeRoute: RouteCreator =
       ).data.logout_url || ""
 
     res.render("welcome", {
-      session: session
-        ? JSON.stringify(session, null, 2)
-        : `No valid Ory Session was found.
-Please sign in to receive one.`,
-      hasSession: Boolean(session),
-      logoutUrl,
+      layout: "welcome",
+      nav: navigationMenu(session, logoutUrl, "welcome"),
+      concepts: [
+        CardGradient({
+          heading: "Getting Started",
+          content:
+            "Jump start your project and complete the quickstart tutorial to get a broader overview of Ory Cloud.",
+          action: "/",
```

Comment:
We still need the correct URLs here, right?

Reply:
yes :)